CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 ### Added
 
 * certifi dxpy dependency
+* docker caching to the CLI commands
```

Comment:
`--cache-docker` for `dx build --nextflow`?

Reply:
sure, in [403e1ba](https://github.com/dnanexus/dx-toolkit/pull/1280/commits/403e1ba3f83672c24cbb1c97901f4aae7016a7b6)